src/components/organisms/DropdownContent/DropdownContent.tsx

```diff
@@ -0,0 +1,151 @@
+import { Dispatch, FC, SetStateAction, useState } from 'react'
+import classNames from 'classnames'
+import { includes, map } from 'lodash'
+import CheckBoxInput from 'components/molecules/CheckBoxInput/CheckBoxInput'
+import Button, {
+  AppearanceTypes,
+  SizeTypes,
+} from 'components/molecules/Button/Button'
+import { SelectionControlsInputProps } from 'components/organisms/SelectionControlsInput/SelectionControlsInput'
+
+import classes from './styles.module.scss'
+
+type DropdownContentProps = SelectionControlsInputProps & {
+  isOpen?: boolean
+  selectedOptionObjects?: { label: string; value: string }[]
+  setIsOpen?: Dispatch<SetStateAction<boolean>>
+}
+
+const DropdownContent: FC<DropdownContentProps> = ({
+  dropdownSize = 'l',
+  disabled,
+  isOpen,
+  searchInput,
+  options,
+  multiple = false,
+  value,
+  name,
+  ariaLabel,
+  buttons = false,
+  cancelButtonLabel,
+  proceedButtonLabel,
+  onChange,
+  setIsOpen,
+  helperText,
+  selectedOptionObjects,
+  tags,
+}) => {
+  const [selectedValue, setSelectedValue] = useState<string[]>(
+    value ? (Array.isArray(value) ? value : [value]) : []
+  )
+
+  const handleSingleSelect = (selectedOption: string) => {
+    onChange(selectedOption ? selectedOption : '')
+  }
+
+  const handleMultipleSelect = (selectedOption: string) => {
+    const selectedValues = Array.isArray(selectedValue)
+      ? [...selectedValue]
+      : []
+    const optionIndex = selectedValues.indexOf(selectedOption)
+
+    const newSelectedValues =
+      optionIndex === -1
+        ? [...selectedValues, selectedOption]
+        : selectedValues.filter((value) => value !== selectedOption)
+
+    setSelectedValue(newSelectedValues)
```

Comment:
**[non-blocker]** 

Not important, but would suggest using lodash here to reduce the code inside this function.
Essentially here you are checking if the string (selectedOption) is inside the array values (selectedValues).

1. Array.isArray(selectedValue) - This check should not be necessary, typescript already says that this has to be an array of strings, so there shouldn't be any possibility that this isn't an array.

2. Currently you are using: const optionIndex = selectedValues.indexOf(selectedOption) + optionIndex === -1 to determine, if the value is already represented in the array or not. This is fine, but you could also use "includes" from lodash instead, to make this slightly shorter

3. Currently you are using filter for the array to remove the value. This is also fine, but would be slightly shorter with lodash without method.